RESYNC.md

```diff
@@ -49,29 +49,31 @@ edges to the existing references.
 ### Default folder structure
 
 ```
-📦 resync_v1
+📦 resync
 ├─ fornebu
 │  ├─ data_model_configuration.yaml
-│  ├─ bid_configuration_day_ahead.yaml
 │  ├─ water_value_based_partial_bid_configuration.yaml
 │  ├─ shop_based_partial_bid_configuration.yaml
 │  ├─ generator.yaml
 │  └─ ... (all types)
-├─ shared
+├─ stavanger
 │  ├─ data_model_configuration.yaml
+│  ├─ water_value_based_partial_bid_configuration.yaml
+│  ├─ shop_based_partial_bid_configuration.yaml
+│  ├─ generator.yaml
+│  └─ ... (all types)
+├─ shared
+│  ├─ bid_configuration_day_ahead.yaml
 │  ├─ price_area_information.yaml
 │  ├─ market_configuration.yaml
 │  ├─ shop_commands.yaml
 │  └─ ... (all types)
-├─ files
-│  ├─ model.yaml
-│  └─ other_files.yaml
-└─ resync_configuration.yaml
+└─ configuration.yaml
 ```
 
 [!WARNING]
 All data model files need to be located in subfolders from the root folder that is specified inside
-the `resync_configuration.yaml` in the `working_directory` field.
+the `configuration.yaml` in the `working_directory` field.
```

Comment:
what became of `working_directory`?

Reply:
The working directory has stayed the same and is still used to find all the subfolders of configs to process. For this repo it's defined as `working_directory: resync` in the `configuration.yaml` file